src/config/mod.rs

```diff
@@ -374,6 +374,7 @@ macro_rules! cf_config {
             pub checksum: ChecksumType,
             #[online_config(skip)]
             pub max_compactions: u32,
+            pub enable_partition_filter: Option<bool>,
```

Comment:
What's the option for? Seems unnecessary if the default value is false.

Reply:
> What's the option for? Seems unnecessary if the default value is false.

We would make it dynamic according to different environment or settings, such as when the data size is large, use partition filter, etc.  For now we don't know the best practice of using it yet, so make it default false.